packages/synapse-react-client/src/components/doi/CreateOrUpdateDoiModal.tsx

```diff
@@ -232,6 +254,14 @@ export function CreateOrUpdateDoiModal(props: CreateOrUpdateDoiModalProps) {
         </Link>
         .
       </Typography>
+      {portalId && (
+        <TextField
+          label={'Publisher'}
+          fullWidth
+          value={portal?.name || null}
+          disabled
+        />
+      )}
```

Comment:
The `publisher` will be the portal name (set by the backend) -- show this in a disabled text field to the user